src/plugins/query_enhancements/server/search/sql_search_strategy.ts

```diff
@@ -33,6 +33,7 @@ export const sqlSearchStrategyProvider = (
   return {
     search: async (context, request: any, options) => {
       try {
+        request.body.fetch_size = await context.core.uiSettings.client.get('discover:sampleSize');
```

Comment:
nit: instead of modifying objects directly, we should try to spread them like:
const newObj = { ...oldObj, newStuff }

It's a small thing but helps avoid unexpected side effects 👍
```

This version:
- Keeps it casual but informative
- Includes a quick example
- Gives a brief reason why
- Uses emoji to keep it friendly
- Still marks it as a "nit" to show it's a minor suggestion

Reply:
changed